scripts/check-public-api.spec.ts

```diff
@@ -168,6 +169,24 @@ describe('check-public-api', () => {
       mock.restore();
     });
 
+    it('parses exports types correctly',async ()=>{
```

Comment:
[q] Is this related to the regex rule?

Reply:
no I thought the error was related to export type so I added this extra test in the hope to reproduce the issue. I thought it is good to keep if we change our parsing to provide some extra safety.